src/Voron/StorageEnvironmentOptions.cs

```diff
@@ -207,9 +210,24 @@ public void SetCatastrophicFailure(ExceptionDispatchInfo exception)
             _catastrophicFailureNotification.RaiseNotificationOnce(exception.SourceException);
         }
 
+        public bool IsCatastrophicFailureSet => _catastrophicFailure != null;
```

Comment:
Why is this an instance method exposing a static field?

Reply:
Static field? `_catastrophicFailure` isn't static

Reply:
Never mind, I read it as `_skipCatastrophicFailureAssertion` somehow

Reply:
ok